### PR TITLE
build(poetry): add launcher script 

### PR DESCRIPTION
**Pull request by wizard-28**
## Purpose

~Adds a launcher to launch the pacstall program,~ Adds a poetry script to launch Pacstall. So we could keep the imports sane for packaging later on.

Use `sudo ~/.local/bin/poetry run pacstall` to run pacstall.

## Approach

~Add `launcher.py`~ Add poetry script, and refactor imports to point to the `pacstall` package

## Progress

- [x] ~Add `launcher.py` script~ Add poetry script
- [x] Move `__main__.py` code to `main()` function
- [x] Refactor imports

## Checklist

- [x] I confirm that I have read the [contributing guidelines](https://github.com/pacstall/pacstall/blob/develop/CONTRIBUTING.md), and this pull request is abiding by all the clauses stated in the guideline.

### CODE REVIEW

**Review by Henryws — APPROVED** [review]

**Comment by wizard-28** [comment]
Waiting for approval by @Henryws, @D-Brox

**Review by Henryws — APPROVED** [review]

**Review by D-Brox — APPROVED** [review]

**Pull request merged**
